model/modules/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .attention import LuongAttention
```

Comment:
What about LocalAttention?
It seems that we should import all attention layers or not import them at all.